### ISSUE TITLE
Weather and Temp. tags from front end?

### ISSUE THREAD

**Issue by S-Farooq** [M-Anwar/ARGEL#6]
I think it is best if front end sends these tags in specific strings for now. The specific string format currently set-up for weather are 'sunny', 'cloudy', 'rainy', 'snowy', or 'N/A' and any other ones will have to be added manually since a conversion to numerics needs to take place.  For temp. the formats are '(_some number, can be -ve_)(F/C)' so ex. '-20C', '50F'.

**Comment by M-Anwar** [comment]
Yes we can work on this for the demo, right now there is no front end for specifying the weather and temperature, but you can setup what ever format you want and when the front end on the website becomes available I can format it to work.
